packages/renderer/src/stores/kubernetes-context-health.ts

```diff
@@ -0,0 +1,31 @@
+/**********************************************************************
+ * Copyright (C) 2024 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import { readable } from 'svelte/store';
+
+import type { ContextHealth } from '/@api/kubernetes-contexts-healths';
+
+export const kubernetesContextsHealths = readable<ContextHealth[]>([], set => {
+  window.events?.receive('kubernetes-contexts-healths', (value: unknown) => {
+    set(value as ContextHealth[]);
+  });
+  window
+    .kubernetesGetContextsHealths()
+    .then(value => set(value))
+    .catch((err: unknown) => console.log('Error getting contexts healths', err));
+});
```

Comment:
question: why this store is not using the new EventStore pattern so we can monitor and track the calls ?

Reply:
I considered using it, but I hardly measure the cost of using it.

I'll look more deeply at it

Reply:
ok, I replaced with an EventStore